code/game/machinery/walllockers.dm

```diff
@@ -1,33 +0,0 @@
-/obj/structure/walllocker
-	name = "Wall Locker"
-	icon = 'icons/obj/lockwall.dmi'
-	icon_state = "emerg"
-	var/list/spawnitems = list()
-	anchored = 1
-	var/amount = 3 // spawns each items X times.
```

Comment:
The old version of the file I changed below. Before, the lockers would spawn the contents 3 times instead of having just 1 set to pull out and put in.